packages/lesswrong/components/search/TagsSearchHit.tsx

```diff
@@ -29,8 +39,12 @@ const TagsSearchHit = ({hit, clickAction, classes}: {
   clickAction?: any,
   classes: ClassesType,
 }) => {
+  const { LWTooltip } = Components
   const tag = hit as AlgoliaTag;
   return <div className={classes.root}>
+    <LWTooltip title="Tag">
```

Comment:
The EA Forum calls these "Topics" now, so you'll need to use the new config that JP created:

`taggingNameIsSet.get() ? taggingNameCapitalSetting.get() : 'Tag'`

Reply:
You can just use taggingNameCapitalSetting.get() as it defaults to Tag

Reply:
Oh right, I forgot